featuregates/controller/go.mod

```diff
@@ -2,23 +2,16 @@ module github.com/vmware-tanzu/tanzu-framework/featuregates/controller
 
 go 1.18
 
-replace (
-	github.com/vmware-tanzu/tanzu-framework/apis/cli => ../../apis/cli
-	github.com/vmware-tanzu/tanzu-framework/apis/config => ../../apis/config
-	github.com/vmware-tanzu/tanzu-framework/apis/core => ../../apis/core
-	github.com/vmware-tanzu/tanzu-framework/cli/runtime => ../../cli/runtime
-	github.com/vmware-tanzu/tanzu-framework/featuregates/client => ../client
-	github.com/vmware-tanzu/tanzu-framework/util => ../../util
-)
+replace github.com/vmware-tanzu/tanzu-framework/util => github.com/vmware-tanzu/tanzu-framework/util v0.0.0-20230119181514-3c34115bc248
 
 require (
 	github.com/go-logr/logr v1.2.3
 	github.com/onsi/ginkgo v1.16.5
 	github.com/onsi/gomega v1.20.2
-	github.com/vmware-tanzu/tanzu-framework/apis/config v0.0.0-20220824221239-af5a644ffef7
-	github.com/vmware-tanzu/tanzu-framework/apis/core v0.0.0-00010101000000-000000000000
-	github.com/vmware-tanzu/tanzu-framework/cli/runtime v0.0.0-00010101000000-000000000000
-	github.com/vmware-tanzu/tanzu-framework/featuregates/client v0.0.0-20221024130358-59eae49d96aa
+	github.com/vmware-tanzu/tanzu-framework/apis/config v0.0.0-20230302191204-eb0dfd6919e5
```

Comment:
This makes the controller depend on older version of APIs. If we need to make an API change, the API change has to be check-in first; followed by the controller change check-in. Do we aware of this limitation? Is it possible to make the build tooling to use the config from main itself?

Reply:
Since current version of build tooling scripts only use the specified "COMPONENT" directory for builds, resolving the relative paths specified with `replace` directive fails during `go mod download` operation. Thus, replaced relative references with latest available package versions.